packages/backend/src/persistence/storage/postgres/doesAccountExist.ts

```diff
@@ -0,0 +1,17 @@
+import { PrismaClient } from '@prisma/client';
+
+export const doesAccountExist =
```

Comment:
what is the benefit of using doesAccountExist over !!getAccount()

Reply:
this is used in lib shared or something, too, and delivery-service and backend now have different return types on getAccount, which causes trouble. Introducing this extra function solves that.